File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    list = s.lower().split(' ')
    
    for word in list:
        new_word = ''
        for i in range(len(word)):
            if i % 2 == 0:
                new_word = new_word + word[i].upper()
            else:
                new_word = new_word + word[i]
        list[list.index(word)] = new_word
    answer = ' '.join(list)
    return answer

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    let words = s.split(' '); 

    for (let i = 0; i < words.length; i++) {
        let word = words[i].split('');
        for (let j = 0; j < word.length; j++) {
            if (j % 2 === 0) {
                word[j] = word[j].toUpperCase();  
            } else {
                word[j] = word[j].toLowerCase();  
            }
        }
        words[i] = word.join('');  
    }

    answer = words.join(' '); 
    return answer;
}
